content/en/docs/releases/supported-releases/index.md

```diff
@@ -51,15 +51,21 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
-| Version         | Currently Supported   | Release Date        | End of Life          | Supported Kubernetes Versions | Untested, but may work on Kubernetes Versions |
-|-----------------|-----------------------|---------------------|----------------------|-------------------------------|-----------------------------------------------|
-| master          | No, development only  |                     |                      |                               |                                               |
-| 1.10            | Yes                   | May 18, 2021        | ~Nov 2021 (Expected) | 1.18, 1.19, 1.20, 1.21        | 1.16, 1.17                                    |
-| 1.9             | Yes                   | February 9, 2021    | Aug 18, 2021         | 1.17, 1.18, 1.19, 1.20        | 1.15, 1.16                                    |
-| 1.8             | Yes                   | November 10, 2020   | May 12, 2021         | 1.16, 1.17, 1.18, 1.19        | 1.15                                          |
-| 1.7             | No                    | August 21, 2020     | Feb 25, 2021         | 1.16, 1.17, 1.18              | 1.15                                          |
-| 1.6             | No                    | May 21, 2020        | November 23, 2020    | 1.15, 1.16, 1.17, 1.18        |                                               |
-| 1.5 and earlier | No                    |                     |                      |                               |                                               |
+| Version         | Currently Supported   | Release Date        | End of Life          | Supported Kubernetes Versions | Untested, but may work |
+|-----------------|-----------------------|---------------------|----------------------|-------------------------------|------------------------|
+| master          | No, development only  |                     |                      |                               |                        |
+| 1.11            | Yes                   | August 2021 (Expect)| ~Feb 2022 (Expected) | 1.19, 1.20, 1.21, 1.22        | 1.16, 1.17, 1.18       |
+| 1.10            | Yes                   | May 18, 2021        | ~Nov 2021 (Expected) | 1.18, 1.19, 1.20, 1.21        | 1.16, 1.17             |
+| 1.9             | Yes                   | February 9, 2021    | Aug 18, 2021         | 1.17, 1.18, 1.19, 1.20        | 1.15, 1.16             |
+| 1.8             | No                    | November 10, 2020   | May 12, 2021         | 1.16, 1.17, 1.18, 1.19        | 1.15                   |
+| 1.7             | No                    | August 21, 2020     | Feb 25, 2021         | 1.16, 1.17, 1.18              | 1.15                   |
+| 1.6             | No                    | May 21, 2020        | November 23, 2020    | 1.15, 1.16, 1.17, 1.18        |                        |
+| 1.5 and earlier | No                    |                     |                      |                               |                        |
```

Comment:
Why do we decide to draw the line here? Should it be 1.8 and earlier?

Reply:
That's the same comment I made above with moving the cutoff up to 1.6 (as opposed to your 1.8). Will I think 1.8 would be the appropriate thing to do (keep the last line being a single `unsupported` line, I know we have many users on older versions and having the supported k8s versions in the table may make sense. Not sure if @therealmitchconnors has a suggestion on where the majority of users are.

Reply:
istr asking @jacob-delgado about this once?